script.js

```diff
@@ -0,0 +1,61 @@
+const bookList = document.getElementById('book-list');
+const addBookForm = document.getElementById('add-book-form');
+
+// Load the books from localStorage or use an empty array
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+function displayBooks() {
+  // Clear the current book list
+  bookList.innerHTML = '';
+
+  // Add each book to the list
+  books.forEach((book, index) => {
+    const li = document.createElement('li');
+    li.innerText = `${book.title} by ${book.author}`;
+    bookList.appendChild(li);
```

Comment:
- [ ] Please make sure to have your app running properly. Books are not being added due to an error with the bookList
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/37901469/226512310-125c8cd4-8997-4810-bc7f-e427612266de.png">

Reply:
@m05e5 Thanks. I highly appreciate  your feedback provided at this point. I'll work on those suggestions then.